numba/cuda/tests/cudasim/test_cudasim_issues.py

```diff
@@ -1,14 +1,49 @@
 import threading
 
 import numpy as np
+from numpy import float64
```

Comment:
Convention is to use `float64` from the `np` module directly in code e.g. `np.array([1,], dtype=np.float64)`, Numba also has a `numba.float64`.